describe/describePrivate.ts

```diff
@@ -0,0 +1,5 @@
+export type PrivateDescription = string;
+
+export function describePrivate(element: Element): PrivateDescription {
+  return element.outerHTML;
```

Comment:
What do you think about adding a comment to the effect of

```suggestion
  // TODO(#14): canonicalize the XML string
  return element.outerHTML;
```

Reply:
Good idea thank you.